internal/provider/deployment.go

```diff
@@ -254,7 +254,7 @@ func (d *DeploymentDeployer) syncContract(sub subi.SubstrateExt) error {
 	if d.Id == "" {
```

Comment:
let's export this one as it's part of the Syncer interface now

Reply:
It is not